guide/how-it-works/lightning-services.md

```diff
@@ -0,0 +1,325 @@
+   alt-text = "Graphic depicting examples of lightning addresses"
+   width = 800
+   height = 348
+%}
+
+### Receive payments offline
+
+Non-custodial lightning wallets need to be online to receive payments. [Daily spending wallets]({{ '/guide/daily-spending-wallet/' | relative_url }}) in particular are regularly offline so receiving payments can be difficult.
+
+LWS can act as third parties that non-custodially hold onto a payment and forward it to the user once they come online. A product that offers this for users is [Greenlight](https://blockstream.com/lightning/greenlight/).
+
+This service is usually required to be offered alongside an LSP. Read more about this [here](https://medium.com/breez-technology/get-ready-for-a-fresh-breez-multiple-apps-one-node-optimal-ux-519c4daf2536).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Offline-payment.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Offline-payment-mobile@2x.png"
+   alt-text = "Graphic depicting how an LWS helps users accept payments whilst offline"
+   width = 800
+   height = 348
+%}
+
+### Swaps
+
+Moving bitcoin between lightning and on-chain can be costly and complicated if done by opening and closing payment channels.
+
+LWS can offer [submarine swaps](https://blog.muun.com/a-closer-look-at-submarine-swaps-in-the-lightning-network/) to allow users to easily move bitcoin between lightning and on-chain without having to manage and deal with payment channels.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps.png"
+   retina = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps@2x.png"
+   mobile = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/lightning-services/Submarine-swaps-mobile@2x.png"
+   alt-text = "Graphic depicting how an LWS can conduct swaps moving bitcoin between on-chain and lightning"
+   width = 800
+   height = 348
+%}
+
+### Watchtowers
+
+Users need to regularly be online to monitor their payment channels to prevent cheating attempts. [Daily spending wallets]({{ '/guide/daily-spending-wallet/' | relative_url }}) in particular are regularly offline and so cheating attempts are more likely.
+
+A [watchtower](https://bitcoinops.org/en/topics/watchtowers/) LWS service monitors payment channels of users who are offline and can punish their counterparties if they attempt to steal the users funds.
+
+{% include tip/open.html color="blue" icon="info" label="Using watchtowers" %}
+
+A watchtower should be a different user than the one the user has channels with, such as an LSP, as they have nothing at stake to attempt to cheat the user.
```

Comment:
```suggestion
It's best practice to use a watchtower that is not controlled by the user's LSP. This way, the watchtower has no incentive to cheat the user.
```
Calling the watchtower a "user" brings up the wrong mental image for me. A watchtower could be another user, or it could be more like a service.